src/corelibs/U2View/src/ov_mca/McaEditorWgt.h

```diff
@@ -51,11 +52,11 @@ class U2VIEW_EXPORT McaEditorWgt : public MaEditorWgt {
 
 protected:
     void initActions() override;
-    void initSeqArea(GScrollBar* shBar, GScrollBar* cvBar) override;
-    void initOverviewArea() override;
-    void initNameList(QScrollBar* nhBar) override;
+    void initSeqArea(GScrollBar *shBar, GScrollBar *cvBar) override;
```

Comment:
Please use clang-format file from the repository. I can help you to reformat all files if needed